std-bits/table/src/main/java/org/enso/table/data/column/builder/LongBuilder.java

```diff
@@ -13,31 +12,42 @@
 import org.enso.table.data.column.storage.type.FloatType;
 import org.enso.table.data.column.storage.type.IntegerType;
 import org.enso.table.data.column.storage.type.StorageType;
+import org.enso.table.error.ValueTypeMismatchException;
 import org.enso.table.problems.ProblemAggregator;
 import org.enso.table.util.BitSets;
 
 /** A builder for integer columns. */
-public abstract class LongBuilder extends NumericBuilder {
+public class LongBuilder extends NumericBuilder implements BuilderForLong, BuilderWithRetyping {
   protected final ProblemAggregator problemAggregator;
+  protected long[] data;
 
-  protected LongBuilder(
-      BitSet isNothing, long[] data, int currentSize, ProblemAggregator problemAggregator) {
-    super(isNothing, data, currentSize);
+  protected LongBuilder(int initialSize, ProblemAggregator problemAggregator) {
+    this.data = new long[initialSize];
     this.problemAggregator = problemAggregator;
   }
 
   static LongBuilder make(int initialSize, IntegerType type, ProblemAggregator problemAggregator) {
-    BitSet isNothing = new BitSet();
-    long[] data = new long[initialSize];
     if (type.equals(IntegerType.INT_64)) {
-      return new LongBuilderUnchecked(isNothing, data, 0, problemAggregator);
```

Comment:
I had slight preference for keeping LongBuilder abstract and keeping the class because then in the name of the particular instance you have a clear indication if the builder is Checked or Unchecked.

But okay to keep Unchecked removed. Perhaps then we could rename LongBuilderChecked to BoundCheckedIntegerBuilder? Then it will actually be clear that LongBuilder builds based on Longs (int64) and the 'checked' one is for smaller values.

What do you think? For sure I wanted to make sure that it is very clear which 'mode' we are using here as I imagine it is easy to use wrong one by accident as the differences are a bit subtle.